src/dstack/api/server/_fleets.py

```diff
@@ -73,6 +73,9 @@ def _get_fleet_spec_excludes(fleet_spec: FleetSpec) -> Optional[_ExcludeDict]:
     if not fleet_spec.configuration.reservation:
         configuration_excludes["reservation"] = True
         profile_excludes.add("reservation")
+    if not fleet_spec.configuration.idle_duration:
```

Comment:
Both `None` and `0` are falsey. Shouldn't this check be `is not None`?

Reply:
UPD seems not to be the case, yaml ints are casted to strings, `idle_duration: 0` in YAML → `'0'` in Pydantic model.

Reply:
Fixed